packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/gux-dropdown-multi.tsx

```diff
@@ -347,16 +348,14 @@ export class GuxDropdownMulti {
   }
 
   private getOptionElementByValue(value: string): HTMLGuxOptionElement[] {
-    const listboxOptionElements = Array.from(
+    const listboxOptionElements: HTMLGuxOptionElement[] = Array.from(
       this.root.querySelectorAll('gux-option-multi')
     );
-    const values = value ? value.split(',') : undefined;
-    if (values) {
-      return listboxOptionElements.filter(element =>
-        values.includes(element.value)
-      );
-    }
-    return;
+    const values = value ? value.split(',') : [];
+
+    return listboxOptionElements.filter(element =>
+      values.includes(element.value)
+    );
```

Comment:
Didn't like that this was returning undefined despite return type being array, which caused every call of this method to need to do a null-check and overly complicating things.